packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -0,0 +1,427 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import 'terra-base/lib/baseStyles';
+import Field from 'terra-form/lib/Field';
+import Fieldset from 'terra-form/lib/Fieldset';
+import NumberField from 'terra-form/lib/NumberField';
+import DatePicker from 'terra-date-picker/lib/DatePicker';
```

Comment:
this can be simplified to `import DatePicker from 'terra-date-picker';`

Reply:
Updated to use suggestion.